refactor(cli): added cli and help command

kevsersrca opened the pull request.

kevsersrca committed:
refactor(cli): added cli and help command

ozanh commented on go.mod:
```diff
@@ -0,0 +1,18 @@
+module gomonk
```
Lets change the module name to github.com/ozanh/gomonk

kevsersrca committed:
refactor(cli):
- edited go.mod module name
- added AUTHORS page
- edited LICENSE file according to authors

kevsersrca committed:
refactor(cli):
- added workflow to run go test

kevsersrca committed:
refactor(cli):
- change help command

kevsersrca committed:
refactor(cli):
- edited authors and LICENSE file
